test/simulator/timestamp/call_log_manager/call_log_manager.cpp

```diff
@@ -62,12 +62,20 @@ call_log_manager::process_call_log()
 void
 call_log_manager::process_call_log_entry(json call_log_entry)
 {
-    const std::string method_name = call_log_entry["MethodName"].get<std::string>();
+    const std::string method_name = call_log_entry["method_name"].get<std::string>();
+    std::cout << "Processing entry " << method_name << std::endl;
     switch (_api_map.at(method_name)) {
     case api_method::wiredtiger_open:
-        /* conn = &connection_simulator::get_connection(); */
+        _conn = &connection_simulator::get_connection();
         break;
     case api_method::open_session:
+        session_simulator *session = _conn->open_session();
```

Comment:
Before opening a session, we could check if _conn is defined. If it's null, we should error out.

Reply:
Good point.

Reply:
Fixed.